mdpow/equil.py

```diff
@@ -284,24 +296,47 @@ def topology(self, itp='drug.itp', prm=None, **kwargs):
         dirname = kwargs.pop('dirname', self.BASEDIR('top'))
         self.dirs.topology = realpath(dirname)
 
-        top_template = config.get_template(kwargs.pop('top_template', 'system.top'))
+        settings = {
```

Comment:
This looks like stuff that should not be hidden in `topology()` but elsewhere, especially so that it can be tested separately.

Reply:
Things were simpler previously...